river/Mapping.zig

```diff
@@ -48,3 +53,13 @@ pub fn deinit(self: Self, allocator: *std.mem.Allocator) void {
     for (self.command_args) |arg| allocator.free(arg);
     allocator.free(self.command_args);
 }
+
+/// Returns whether the mapping matches the keysym, modifiers and release status
+pub fn match(self: *const Self, keysym: c.xkb_keysym_t, modifiers: u32, released: bool) bool {
+    return modifiers == self.modifiers and keysym == self.keysym and released == self.release;
+}
+
+/// Returns whether the mappings are euqal
+pub fn isEqual(self: *const Self, other: *const Self) bool {
+    return other.modifiers == self.modifiers and other.keysym == self.keysym;
+}
```

Comment:
I'm not convinced that these functions are a net win for readability. Both are only called in one place and both require the reader to check the function definition to see exactly what is being compared.

(Aside on zig style, these should take `self: Self` as you don't need the pointer, the compiler will optimize this to a pass-by-pointer if it likes)

Reply:
Yeah.
Changed them back.